tests/foreman/cli/test_capsule.py

```diff
@@ -382,3 +386,25 @@ def test_positive_enable_bmc():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.skip_if_not_set('fake_capsules')
+@pytest.mark.tier1
+def test_positive_search_nondefault_proxy(request, target_sat):
+    """Search non-default proxy  with the home proxy
+
+    :id: caf51662-6b4e-11ed-baba-2b9d7b368002
+
+    :expectedresults: Non-default proxy can be searched
+
+    :BZ: 2077824
+
+    :customerscenario: true
+    """
+    proxy = _make_proxy(request, target_sat, options={'name': gen_string('alpha')})
+    path = target_sat.api.Capsule().path() + '?search=id+%21%3D+1'
```

Comment:
Non-blocking, what are we concatenating here with the path? and why do we need that?

Reply:
@adarshdubey-star its a search string used in path to find capsules with id!=1, as mentioned in bz